src/main/kotlin/no/nav/syfo/personstatus/api/v2/model/VeilederHistorikkDTO.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.syfo.personstatus.api.v2.model
+
+import java.time.LocalDate
+
+data class VeilederHistorikkDTO(
```

Comment:
Tror for leselighet at det kan hjelpe å være så spesifikke som vi kan med typen 😊 Dette hadde hjulpet meg å bedre forstå hva typen representerer. Hva tenker du?
```suggestion
data class VeilederTildelingHistorikkDTO(
```